app.py

```diff
@@ -1,53 +1,115 @@
 # -*- coding: utf-8 -*-
 from fastapi import FastAPI, responses, status
 from pydantic import BaseModel
-from typing import List
+from typing import List, Optional, Dict, Any
 from neural_search import util
 from submodules.model.business_objects import general
 
 app = FastAPI()
 
 
-@app.get("/most_similar")
+@app.post("/most_similar")
```

Comment:
just out of interest, is this a post because of needed functionality (the `att_filter`), or is it because of conventions?

Reply:
because a body is needed for att_filter and get doesn't work with a request body :) So yes you are exactly right